conf/application.conf

```diff
@@ -159,3 +159,8 @@ whitelist {
   enabled = false
   eori = []
 }
+
+timeoutDialog {
+  timeout="13min"
+  countdown="2min"
```

Comment:
Is it working in this form? Thought it has to be sth like `13 MINUTES`. Depends on the parser I guess...

Reply:
Yes - seems to work.  I've been testing with 
```
timeoutDialog {
  timeout="130s"
  countdown="2min"
}
```